lib/engine/game/g_system18/meta.rb

```diff
@@ -15,7 +15,7 @@ module Meta
         GAME_LOCATION = 'Various'
         GAME_PUBLISHER = :all_aboard_games
         GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/System18'
-        GAME_RULES_URL = 'https://github.com/tobymao/18xx/wiki/System18'
+        GAME_RULES_URL = 'https://www.dropbox.com/scl/fi/or38t89eczc5eki8e21y7/System18_Rules.pdf'
```

Comment:
The link needs to include the `rkley` parameter (from #10459). That lets you access the file without being signed into Dropbox.
```suggestion
        GAME_RULES_URL = 'https://www.dropbox.com/scl/fi/or38t89eczc5eki8e21y7/System18_Rules.pdf?rlkey=n9kblkl7kvnz96oobs6hsatk2&dl=0'
```